.graphqlconfig

```diff
@@ -1,4 +1,25 @@
 {
```

Comment:
community finally standardized on a a graphql config, so here it is. 
added a github schema so we can query github using graphql, too. no more REST calls.